src/Claim.test.ts

```diff
@@ -284,7 +284,7 @@ describe('Claim.createClaim', async (should: any) => {
       given: 'an extended claim without an external context',
       should: 'will return an Error()',
       actual: claim,
-      expected: new Error('The property "edition" in the input was not defined in the context.'),
+      expected: new Error('The property "edition" in the input was not defined in the context.').message,
```

Comment:
Interesting `expected`. Why not just put the text in?

Reply:
I didn't make this PR to fix the issue, just to alert you so you could fix it in your PR :D

Guess I should have mentioned that lol

Reply:
Like you said though, I think just putting the text in expected and grabbing the message from the error is probably the way to go though

Reply:
I can fix adjust it here really quick so we can merge it in though